### PR TITLE
Init express app

### PR DESCRIPTION
**Pull request by matt-rose6**
- [ ] Clone repo
- [ ] Run npm i
- [ ] Run npm start in command line
- [ ] Navigate to localhost:3001 and check that {"info":"Node.js, Express, and Postgres API"} is present

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const routes = require('./routes');
+const app = express();
+const port = 3001;
```
Comment (nries1) [inline server/index.js:5]: Some modern hosting services (like Heroku) don't deploy from static ports, so you'll need to account for that here. Basically, if the host is using a port already, use that, if not, then you can use 3001. Code would be `const port = process.env.PORT || 3001`

File: server/controllers/businessController.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// const bsService = require('../services/businessServices');
+
+//TODO: any data validation would happen within these classes
+
+const getBusinessList = (request, response) => {
```
Comment (nries1) [inline server/controllers/businessController.js:5]: this could be another PR but you'll probably want to include the next parameter for error handling.

File: server/index.js (hunk 1 above)
Comment (matt-rose6) [reply server/index.js]: made that change thanks

File: server/controllers/businessController.js (hunk 2 above)
Comment (matt-rose6) [reply server/controllers/businessController.js]: added next parameter to each method
